deepreg/dataset/loader/unpaired_loader.py

```diff
@@ -62,22 +62,34 @@ def __init__(
     def validate_data_files(self):
         """Verify all loader have the same files"""
         if self.labeled:
+            image_ids = self.loader_fixed_image.get_data_ids()
+            label_ids = self.loader_fixed_label.get_data_ids()
+            check_difference_between_two_lists(list1=image_ids, list2=label_ids)
+
```

Comment:
Here as the data are not paired, the moving and fixed images come from the same file loader, so no need to check fixed images & labels.

Could you remove these lines and add a comment to explain this point?

Reply:
Yes, that's right